src/main/java/com/stripe/model/Account.java

```diff
@@ -38,6 +38,14 @@
   @Setter(lombok.AccessLevel.NONE)
   ExpandableField<File> businessLogo;
 
+  /**
+   * (ID of a [file upload](https://stripe.com/docs/guides/file-upload)) A larger logo for the
+   * account that will be used in Checkout instead of the smaller logo and without the account's
+   * name next to it if provided. Must be at least 128px x 128px.
+   */
+  @SerializedName("business_logo_large")
+  String businessLogoLarge;
```

Comment:
@remi-stripe  New field appearing, not in binding, but now also in APIref 
https://stripe.com/docs/api/accounts/object#account_object-business_logo_large

Reply:
Yeah we just added it for backwards compatibility reasons. Since go/.net will be on the next API version on 2/19 and java will be on 2/26 I don't think it's a big deal if we don't add it immediately/ignore it.

Reply:
ah okay it will just follow OpenAPI spec doc. When the avedon come I suspect it will be removed into business profile object right?

Reply:
Correct! Just include it. I was lazy and did not do it because the hope is that no one really needs it and since autogen ships soon (and .net/go have the new API version next week) it wasn't worth the chunder of code + PR + release :p